vendor/index.yaml

```diff
@@ -1551,3 +1551,14 @@ vendors:
 
   - id: the-box-development
     name: The-Box Development
+
+  - # Unique identifier of the vendor (lowercase, alphanumeric with dashes, max 36 characters)
+    id: beiselen
+    # Vendor company name
+    name: Beiselen Service GmbH
+    # LoRa Alliance issued Vendor ID
+    vendorID: 845
```

Comment:
Did you guys join the LoRa Alliance recently? I don't see you in the new members list.